tests/usbredir_test.go

```diff
@@ -175,10 +177,17 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 	})
 })
 
-func createVMI(virtClient kubecli.KubevirtClient, enableUsbredir bool) (*v1.VirtualMachineInstance, error) {
-	randomVmi := tests.NewRandomVMI()
-	if enableUsbredir {
-		randomVmi.Spec.Domain.Devices.ClientPassthrough = &v1.ClientPassthroughDevices{}
-	}
+func createVMIWithUsbredirEnabled(virtClient kubecli.KubevirtClient) (*v1.VirtualMachineInstance, error) {
+	randomVmi := libvmi.New(
+		libvmi.WithResourceMemory("1Mi"),
```

Comment:
In which scenarios this is needed?
Why not include it within `libvmi.New()`?

Reply:
As you said it is out of the scope of this PR if it's indeed need to be done.